rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/widget/OneScanCovidRDTBarcodeFactoryTest.java

```diff
@@ -99,6 +100,10 @@ public void testOnActivityResultShouldPopulateCorrectData() throws Exception {
         });
         RDTJsonFormUtilsShadow.setJsonObject(deviceDetailsWidget);
 
+        String displayValue = String.format("%s,%s,%s,%s,%s", VAL_0, DATE, VAL_2, VAL_3, SENSOR_TRIGGERED);
+        Barcode barcode = new Barcode();
+        barcode.displayValue = displayValue;
+        intent.putExtra(JsonFormConstants.BARCODE_CONSTANTS.BARCODE_KEY, barcode);
```

Comment:
This is repeated https://github.com/onaio/rdt-standard/pull/656/commits/092b20ae80759d423001e617f8fdbe9ebd5e3987#diff-7c8f82faba856576126f062799a7e505ed0666e6f93cdd2541eec5e2733dd4f0R132

Reply:
fixed